src/app/config/actions.js

```diff
@@ -127,10 +127,10 @@ export function emptyActiveCollection() {
     }
 }
 
-export function updateWorkingOn(id, name) {
+export function updateWorkingOn(id, name, url, collection, error) {
```

Comment:
1. Might be worth considering changing these parameters to one object that has these as properties. If for example, we needed to add a new argument to this function we would be forced to supply, `id`, `name`, `url` `collection` and `error` before our new arguments, or be forced to change the order of the arguments and then every place where these are called. 

e.g. 
```
function updateWorkingOn({id, name, url, collection, error}) {...
```
Which can then be called like `updateWorkingOn({id:a, name:b, error:c})` etc. 

It would just make it easier to maintain and keeps the function open for expansion in the future.

2. I know this was the name of the function already, IMO with just the context of it is in a file named 'actions.js' and function is called `updateWorkingOn` I have no clue what the purpose of this function was without seeing where it was called from. This to me is a sign that annotations are needed, although IMO the whole file should have annotations for each function. If against annotations and opting for self annotating code, then these event names need to be prefixed with something like 'eventUpdateStatusToWorkingOn' (or do both which would be even better).

3. Similar to point 2 but with the argument names. There are no annotations and the names are not declarative enough e.g. 'url' url of what?. This just makes it harder to maintain in the future, and harder for starters to get to grips with the code.

Reply:
Discussed, no action to be taken.